CMakeLists.txt

```diff
@@ -460,7 +460,9 @@ if(BLAS_FOUND)
     -DC_ZAXPY=${C_ZAXPY})
 endif()
 
-if(LAPACK_FOUND)
+#check for functions in Lapack
+#(or OpenBLAS since OpenBLAS typically contains Lapack)
```

Comment:
Why is this change necessary? In my local tests LAPACK is correctly detected with OpenBLAS.

Reply:
On Summit, it is not:

-- BLA_VENDOR=OpenBLAS
-- Looking for Fortran sgemm
-- Looking for Fortran sgemm - found
-- Found BLAS: /sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.3.0/openblas-0.3.17-xre5bzetoh5rctek6hyif6bt4wt2h7c2/lib/libopen                blas.so
-- BLAS_LIBRARIES=/sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.3.0/openblas-0.3.17-xre5bzetoh5rctek6hyif6bt4wt2h7c2/lib/libo                penblas.so
-- LAPACK_LIBRARIES=

Reply:
Interesting. What version `CMake` is running there?

Reply:
cmake version 3.23.2